nf_core/pipelines/create/custompipeline.py

```diff
@@ -23,7 +23,7 @@ def compose(self) -> ComposeResult:
             )
         )
         yield Horizontal(
-            Switch(id="toggle_all", value=True),
+            Switch(id="toggle_all", value=False),
```

Comment:
we should have two kinds of switches, the ones set to true for "opt-out" features, and a different one set to false for "opt-in features"

Reply:
I think one switch makes more sense (from a developers perspective they don't care so much, about what is opt-in or opt-out I think, just if they have everything or not everything). is there a way to show a mixed state in the switch, ie a neutral position?

Reply:
I misunderstood this switch 😄 I have just pushed some changes if you want to have a look! There isn't a way to show a newtral position